ci/helm/environments/unit-tests-mongodb.yaml

```diff
@@ -1,11 +1,16 @@
-# enabled conditions
+# these values are required by helmfile-base.yaml.gotmpl releases
 nuxeo:
   enabled: false
 mongodb:
   enabled: true
 postgresql:
   enabled: false
+opensearch:
+  enabled: false
+kafka:
+  enabled: false
```

Comment:
Should we enable them by the way, to run the unit tests in the same environment as for nuxeo-lts? I saw that Kafka was added to the OpenSearch 2 unit tests: https://github.com/nuxeo/nuxeo-opensearch2/pull/12/files#diff-7e155d5b9fdb38e53e4fa33ab1d3f9c56f04f7ce27a18d85c71c7d8ad6c5a0b9R11 (previously, only MongoDB and OpenSearch were enabled), that's what made me think of it.